src/js/analyses/components/NuVs/NuVsViewer.tsx

```diff
@@ -1,8 +1,8 @@
 import { NuVsList } from "@/analyses/components/NuVs/NuVsList";
+import NuVsToolbar from "@/analyses/components/NuVs/NuVsToolbar";
```

Comment:
Huh I would have though so, but on testing it seems like the answer is no. Might be because NuvsViewer is part of `analyses`, though I don't know why that would matter